models/zeroex/optimism/zeroex_optimism_native_fills.sql

```diff
@@ -57,21 +57,21 @@ WITH
             date_trunc('minute', evt_block_time) = tp.minute and  tp.blockchain = 'optimism'
             AND CASE
                     -- Set Deversifi ETHWrapper to WETH
-                    WHEN fills.takerToken IN ('0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee') THEN '0x4200000000000000000000000000000000000006'
+                    WHEN fills.takerToken IN (0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee) THEN 0x4200000000000000000000000000000000000006
                     ELSE fills.takerToken
                 END = tp.contract_address
         LEFT JOIN {{ source('prices', 'usd') }} mp ON
             DATE_TRUNC('minute', evt_block_time) = mp.minute  and mp.blockchain = 'optimism'
             AND CASE
                     -- Set Deversifi ETHWrapper to WETH
-                    WHEN fills.makerToken IN ('0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee') THEN '0x4200000000000000000000000000000000000006'
+                    WHEN fills.makerToken IN (0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee) THEN 0x4200000000000000000000000000000000000006
                     ELSE fills.makerToken
                 END = mp.contract_address
         LEFT OUTER JOIN {{ ref('tokens_erc20') }} mt ON mt.contract_address = fills.makerToken and mt.blockchain = 'optimism'
         LEFT OUTER JOIN {{ ref('tokens_erc20') }} tt ON tt.contract_address = fills.takerToken and tt.blockchain = 'optimism'
          where 1=1 
                 {% if is_incremental() %}
-                AND evt_block_time >= date_trunc('day', now() - interval '1 week')
+                AND evt_block_time >= date_trunc('day', now() - interval '7' day)
                 {% endif %}
                 {% if not is_incremental() %}
                 AND evt_block_time >= '{{zeroex_v3_start_date}}'
```

Comment:
Add missing timestamp.
```suggestion
                AND evt_block_time >= TIMESTAMP '{{zeroex_v3_start_date}}'
```